### DISCUSSION TITLE
Document could not be loaded

### DISCUSSION THREAD

**Discussion by skills-up** [dedoc/scramble#118] [Q&A] [answered]
Getting this error while trying to visit `/docs/api`

> The API description document could not be fetched. This could indicate connectivity problems, or issues with the server hosting the spec.

NGINX log shows this:

> open() "/docs/api.json" failed (2: No such file or directory)

However, when I visit `/docs/api.json` then the JSON is visible alright.

How do I resolve this error?

**Comment by romalytvynenko** [comment, marked as answer]
Hey.

Seems like something is off with nginx config. Any chance you can create repro repo (with docker)?
